internal/devices/devices.go

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2025 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package devices
+
+const (
+	DevType1EnvVar = "E2E_NROP_DEVICE_TYPE_1"
+	DevType2EnvVar = "E2E_NROP_DEVICE_TYPE_2"
+	DevType3EnvVar = "E2E_NROP_DEVICE_TYPE_3"
+)
```

Comment:
what's the benefit of moving here the constants?

Reply:
if they were kept in `test/` we cannot use them in the `internal/` directory because of cycle imports because `test/` directory already imports `internal/`. to solve this we move the common consts in internal and use them in `test/` and internal helpers without issue. I believe it would help to have this change in separate commit. I'll split that.

Reply:
I've grown less and less entusiast of constants-only packages (and yes, I'm guilty too) but this makes sense.